packages/sampling/src/sampler.ts

```diff
@@ -187,6 +184,19 @@ export abstract class OboeSampler implements Sampler {
       }
     }
 
+    if (!s.settings) {
+      this.logger.debug("settings unavailable; sampling disabled")
+
+      if (s.traceOptions?.triggerTrace) {
```

Comment:
Do we still need to check trigger trace here?

Reply:
Yes, this is to send `X-Trace-Options-Response: trigger-trace=settings-not-available` (see row 4 in [trigger trace spec matrix](https://swicloud.atlassian.net/wiki/spaces/NIT/pages/2770436395/Trigger+Trace#Request%2FResponse-Matrix))